src/common/data.s

```diff
@@ -81,5 +82,6 @@ _is_running_sim:        .byte $01
 _current_step:          .byte $ff
 _info_display_count:    .byte $00
 _is_darkmode:           .byte $00
+_is_showing_info:       .byte $01
```

Comment:
I guessed at this assuming `$01` is setting the item to `true` (`1`).

Reply:
it is